src/test/java/tec/uom/se/format/SymbolMapTest.java

```diff
@@ -52,7 +52,7 @@
    */
   @Test
   public void testSymbolMap() {
-    SymbolMap symbols = SymbolMap.of(ResourceBundle.getBundle(BUNDLE_NAME, Locale.getDefault()));
+    SymbolMap symbols = SymbolMap.of(ResourceBundle.getBundle(BUNDLE_NAME, Locale.ROOT));

```

Comment:
Ah, that's a very good use of `Locale.ROOT`. You're basically saying not to use locale-specific overrides --- you just want to use the symbols in the base resource bundles.